command/pr.go

```diff
@@ -63,9 +64,11 @@ $ gh pr view --web`,
 var prListCmd = &cobra.Command{
 	Use:   "list",
 	Short: "List and filter pull requests in this repository",
-	Example: `$ gh pr list --limit all
-$ gh pr list --state closed
-$ gh pr list --label “priority 1” “bug”`,
```

Comment:
1. This was incorrect syntax for multiple labels
2. Curly quotes are not supported in shells, so I've replaced them with normal double quotes

Reply:
This is what I get for copy/pasting directly from a google doc 😞